### DISCUSSION TITLE
run all templates in passive mode

### DISCUSSION THREAD

**Discussion by savushkin-yauheni** [projectdiscovery/nuclei-templates#1287] [Q&A] [answered]
Hi team.

Is there any way to run all templates in passive mode and execute only matchers on response ?

I tried and get a lot of errors. I think if  path is not equal `"{{BaseURL}}"` it doesn't work ?

errors example:
```
[WRN] Could not parse file '/root/nuclei-templates/technologies/opencast-detect.yaml': cannot create template executer 
```

steps to reproduce:

1. create file in folder `test`
```
HTTP/1.1 200 OK

test
```

2. `echo test | nuclei -t ~/nuclei-templates/ -passive -debug`

Thanks!

**Comment by ehsandeep** [comment, marked as answer]
From:- https://blog.projectdiscovery.io/nuclei-v2-3-0-release/

> Passive HTTP Support has been added. This allows available Nuclei HTTP templates (**limited to templates for root paths**) to be run on the passively collected data (from tools like httpx, meg, etc) and perform post-processing on them! This feature is available with -passive flag in Nuclei.

So initially we intended to support all templates and later found out unexpected matches, as we have passive data collected for root path, allowing templates for different path doesn't make sense and get unexpected matches for template having path against response data of root path, so this support is limited for the template having `"{{BaseURL}}"` as input.
